tekton/pipeline_run.go

```diff
@@ -120,6 +122,18 @@ func (r *ReleasePipelineRun) WithApplicationSnapshot(snapshot *applicationapiv1a
 	return r
 }
 
+//  WithEnterpriseContractPolicy adds a param containing the EnterpriseContractPolicy Spec as a json string to the release PipelineRun.
+func (r *ReleasePipelineRun) WithEnterpriseContractPolicy(ecpolicy *ecapiv1alpha1.EnterpriseContractPolicy) *ReleasePipelineRun {
+	ecPolicy, _ := json.Marshal(ecpolicy.Spec)
+
+	r.WithExtraParam("policy", tektonv1beta1.ArrayOrString{
```

Comment:
Do not hardcode the name. Do the same we do with the ApplicationSnapshot and change the Bundle.

Reply:
changed to use the ec policy kind.